site/docs/plugins/fluent.md

```diff
@@ -0,0 +1,155 @@
+# Internationalization with Fluent
+
+[Fluent](https://projectfluent.org/) is a localization system made by the Mozilla Foundation for natural-sounding translations.
+It has a very powerful and elegant syntax that lets anyone write efficient and fully-understandable translations.
+This plugin takes advantage of this amazing localization system to make grammY-powered bots fluent with high-quality translations.
+
+## Instantiate Fluent
+
+The next step would be to create a Fluent instance:
+
+```typescript
+import { Fluent } from "@moebius/fluent";
```

Comment:
This is outdated, no?

Reply:
No it is not. The `Fluent` class is from the `@moebius/fluent` package.

Reply:
Ohhh I see, my bad.